Vehicle.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeReviewDemo
+{
```

Comment:
The property `model` should follow PascalCase naming conventions. Rename it to `Model`.